src/stories/components/select-item/select-item.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import CheckboxOff from '../svg/checkbox-off';
+import CheckboxOn from '../svg/checkbox-on';
+
+interface SelectItemProps {
+  label: string | JSX.Element;
+  count?: number;
+  checked?: boolean;
+  onClick?: () => void;
+  minWidth?: number;
+}
+
+export const SelectItem = ({ checked = false, minWidth = 0, ...props }: SelectItemProps) => {
+  const [focused, setFocused] = useState(false);
+
+  return <Container className="no-select" onClick={props.onClick} minWidth={minWidth}>
+    {!checked && <CheckboxOff style={{ padding: '2px' }} fill={focused ? '#708390' : '#9FAFB9'}/>}
```

Comment:
Just and ide, what do you thing to use ternary operator in here, give me your feedback about what do you think